CHANGELOG.md

```diff
@@ -138,6 +138,7 @@ All notable, unreleased changes to this project will be documented in this file.
   - drop `created` argument from `draftOrders` query
   - drop `productType` from `ProductFilter`
   - deprecate mutations' `<name>Errors`, typed `errors` fields and remove deprecation
+- Add channel data to Order webhook - #7299 by @krzysztofwolski
```

Comment:
This change is not breaking. Also it improves the feature that is new in 3.0 so maybe it doesn't have to be mentioned in the changelog.